Foundation/NSObjCRuntime.swift

```diff
@@ -228,6 +228,14 @@ internal struct _CFInfo {
     }
 }
 
+internal struct _CFPad {
+#if os(Android)
```

Comment:
I'd want compiler people to chime in on this to understand if the guarantee this code relies on re: sizing will hold. Who could we cc here?